Posts/2021/03/ScriptingGuy.0007.folderexists.md

```diff
@@ -0,0 +1,41 @@
+---
+post_title: Determine if a folder exists
+username: tfl@psp.co.uk
```

Comment:
This should be your username not Thomas's.

Reply:
LOL-he is free to give me credit as I did write much of the post! :-)

Reply:
Resolved ;-)